<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>324</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Century Schoolbook</family>
    <pointsize>8</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Demo LED Arduino</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit_LogMessage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>401</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fixedsys</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_OpenSerial">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Schoolbook</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Open Serial</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Exit">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>260</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Schoolbook</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Lampu_OFF">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Lampu OFF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Lampu_ON">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Lampu ON</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Kipas_ON">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Kipas ON</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Kipas_OFF">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Kipas OFF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Dispenser_ON">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Dispenser ON</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Dispenser_OFF">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>60</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Dispenser OFF</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>422</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
